src/utils/daos_hdlr.c

```diff
@@ -1673,9 +1764,11 @@ fs_copy_dir(struct cmd_args_s *ap,
 
 	/* create the destination directory if it does not exist */
 	rc = file_mkdir(ap, dst_file_dfs, dst_path, &tmp_mode_dir);
-	if (rc != EEXIST && rc != 0)
+	if ((rc == EEXIST) && (strncmp(dst_path, "/", 2) != 0)) {
+		D_DEBUG(DB_TRACE, "Directory %s exists.\n", dst_path);
```

Comment:
I think the error is coming from here, dfs_sys is return EEXIST for '/' as it always exists in a container.  This seems like it would be a problem for any fs copy command though, not just with fault injection on which I don't understand.
```suggestion
	if (rc == EEXIST) {
		if (strncmp(dst_path, "/", 2) != 0) {
			DH_PERROR_SYS(ap, rc, "Directory '%s' exists", dst_path);
		}
```